go/teams/loader2.go

```diff
@@ -715,6 +726,15 @@ func (l *TeamLoader) addSecrets(mctx libkb.MetaContext,
 	readerKeyMasks []keybase1.ReaderKeyMask) error {
 
 	state := team.MainChain()
+	stateWrapper := newTeamSigChainState(team)
+	role, err := stateWrapper.GetUserRole(me)
+	if err != nil {
+		role = keybase1.TeamRole_NONE
+	}
+	// Bots don't have any team secrets, so we short circuit.
+	if role.IsBot() {
+		return nil
+	}
```

Comment:
This might make more sense in loader.go at the callsite of addSecrets. Assuming `computeSeedChecks` and `addKBFSCryptKeys` should also be skipped. Also, bots don't have secrets, could you remind what they _do_ have that enables them to communicate?

Reply:
non-bot team members derive a key from the team ephemeral key and box it for the bot's user ephemeral key: https://github.com/keybase/client/blob/master/go/ephemeral/teambot_ek.go#L22-L27. Conversations involving the bot use this derived key and then members pipe the bot's responses back into a normal team conv